instructor/templates/instructor/assignment_sidebar.html

```diff
@@ -21,27 +23,31 @@
         </div>
 
         <div class='scb_ab_s_sidebar_section_heading' role='heading'>Experiment Setup</div>
-        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "strains" %}scb_s_sidebar_page_name_selected{% endif %}'>
             <a class="scb_s_assignment_sidebar_link"
                href="{% if not new %}{% url "common_assignments_edit_strains" %}{% endif %}">
                 Strains
             </a>
         </div>
-        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+            {% if page_name == "variables" %}scb_s_sidebar_page_name_selected{% endif %}'>
             <a class='scb_s_assignment_sidebar_link'
                href="{% if not new %}{% url "common_assignments_variables" %}{% endif %}">
                 Select variables
             </a>
         </div>
 
-        <div role="link" class='scb_s_instructor_sidebar_link_heading '>
+        <div role="link" class='scb_ab_s_sidebar_page_link
+        {% if page_name == "treatments" %}scb_s_sidebar_page_name_selected{% endif %}'>
```

Comment:
It looks like this CSS class is defined starting with `scb_ab_s_...`. As it is I can't see the highlight on the panel